sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Constants.cs

```diff
@@ -35,6 +35,12 @@ public class ServiceEnvironmentVariable
     /// The environment variable for the Playwright service run ID.
     /// </summary>
     public static readonly string PlaywrightServiceRunId = "PLAYWRIGHT_SERVICE_RUN_ID";
+
+    /// <summary>
+    /// The environment variable for the Playwright service one time operation.
+    /// </summary>
+
+    public static readonly string OneTimeOperationFalg= "ONE_TIME_OPERATION_FLAG";
```

Comment:
Make this internal

Reply:
Add it here - https://github.com/Azure/azure-sdk-for-net/blob/05460f0efbc3dc34c9bbcb4aa5d4a7200b3612b5/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Constants.cs#L223

Reply:
Ideally it should be prefixed with _MPT. We will need to update it in the JS package as well later.